pom.xml

```diff
@@ -365,5 +366,44 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- TODO remove with parent pom or spotless plugin update https://github.com/diffplug/spotless/issues/1774 -->
+    <profile>
+      <id>java11+</id>
+      <activation>
+        <jdk>[11,)</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.diffplug.spotless</groupId>
+            <artifactId>spotless-maven-plugin</artifactId>
+            <version>2.39.0</version>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>java21+</id>
+      <activation>
+        <jdk>[21,)</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.diffplug.spotless</groupId>
+            <artifactId>spotless-maven-plugin</artifactId>
+            <configuration>
+              <java>
+                <palantirJavaFormat>
+                  <version>2.35.0</version>
+                </palantirJavaFormat>
+              </java>
+            </configuration>
```

Comment:
Should be done also in parent 😄

Reply:
I'm waiting for the sportless plugin will be released with the complete fix. Now just experimenting locally only.